### ISSUE TITLE
An error

### ISSUE THREAD

**Issue by camCX** [ivyha010/AttendAffectNet#1]
Hi, 

Maybe there is an obvious error in your model code:

`src = torch.cat((reduced1, reduced2, reduced3, reduced4, reduced5), dim=1)` should be `src=torch.stack((reduced1, reduced2, reduced3, reduced4, reduced5), dim=1)`
for
`#  BASE: ENCODER and a FULLY CONNECTED LAYER
class Encoder_FullyConnected(nn.Module):
    """
    A standard Encoder-Decoder architecture. Base for this and many
    other models.
    """
    def __init__(self, inputdim1, inputdim2, inputdim3, inputdim4, inputdim5, src_embed, encoder, d_model= 8):
        super(Encoder_FullyConnected, self).__init__()
        print('dims: ', inputdim1, inputdim2, inputdim3, inputdim4, inputdim5)
        self.linear1 = nn.Linear(inputdim1, d_model)
        self.linear2 = nn.Linear(inputdim2, d_model)
        self.linear3 = nn.Linear(inputdim3, d_model)
        self.linear4 = nn.Linear(inputdim4, d_model)
        self.linear5 = nn.Linear(inputdim5, d_model)

        self.src_embed = src_embed
        self.encoder = encoder
        self.linear = nn.Linear(d_model, 1)
        self.pooling = nn.AvgPool1d(5)  # pooling window = 5

        self.drop = nn.Dropout(0.1)

    def forward(self, src1, src2, src3, src4, src5):
        "Take in and process masked src and target sequences."
        reduced1 = self.linear1(src1)
        reduced2 = self.linear2(src2)
        reduced3 = self.linear3(src3)
        reduced4 = self.linear4(src4)
        reduced5 = self.linear5(src5)

        **src = torch.cat((reduced1, reduced2, reduced3, reduced4, reduced5), dim=1)**
        temp = self.encoder(src)  # shape: batchsize x 5 x d_model
        temp_permute = temp.permute(0, 2, 1)  # shape: batchsize x d_model x 5
        temp2 = self.pooling(temp_permute)  # shape: batchsize x d_model x 1
        temp3 = self.linear(self.drop(temp2.squeeze(-1)))

        return temp3`

Thanks for your work!
